app/javascript/application.js

```diff
@@ -0,0 +1,22 @@
+// Entry point for the build script in your package.json
+import React from 'react';
+import { createRoot } from 'react-dom/client';
+import { BrowserRouter } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import { store } from './context/store';
+import Router from './Router';
+
+function App() {
```

Comment:
- [ ]  Please, kindly refactor this main `App` functional component to ES6 arrow function, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

- [ ] OPTIONAL: Refactor to ES6 there rest of the functions.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry:

![image](https://github.com/BahirHakimy/rent-a-car/assets/57421823/35b9afdc-8386-4fbf-9bb1-6db2b0190931)


That way, your code will meet the latest standards. 💪